hivemind/dht/node.py

```diff
@@ -6,17 +6,19 @@
 from collections import defaultdict, Counter
 from dataclasses import dataclass, field
 from functools import partial
-from typing import Optional, Tuple, List, Dict, DefaultDict, Collection, Union, Set, Awaitable, Callable, Any
+from typing import Any, Awaitable, Callable, Collection, DefaultDict, Dict, List, Optional, Sequence, Set, Tuple, Union
 
+from multiaddr import Multiaddr
 from sortedcontainers import SortedSet
 
 from hivemind.dht.crypto import DHTRecord, RecordValidatorBase
 from hivemind.dht.protocol import DHTProtocol
 from hivemind.dht.routing import DHTID, DHTKey, get_dht_time, DHTValue, BinaryDHTValue, Subkey
 from hivemind.dht.storage import DictionaryDHTValue
 from hivemind.dht.traverse import traverse_dht
-from hivemind.utils import Endpoint, LOCALHOST, MSGPackSerializer, get_logger, SerializerBase, DHTExpiration
-from hivemind.utils.timed_storage import TimedStorage, ValueWithExpiration
+from hivemind.p2p import P2P, PeerID as Endpoint
```

Comment:
I suggest to fix this (replace `Endpoint` to `PeerID` in `dht/{node,protocol,routing}.py`) in a separate PR to avoid exploding this PR with the mechanical changes.

Reply:
Sure, but let's follow up immediately after this one (and possibly after black) then, since we don't want to keep this kind of ambiguity in the repo too long